bubuku/zookeeper.py

```diff
@@ -118,5 +150,143 @@ def take_lock(self, *args, **kwargs):
                 _LOG.error('Failed to obtain lock for exhibitor, retrying', exc_info=e)
 
 
-def load_exhibitor(initial_hosts: list, zookeeper_prefix):
-    return Exhibitor(initial_hosts, 8181, zookeeper_prefix)
+class BukuExhibitor(object):
```

Comment:
I have thoughts about this, because ExhibitorEnsembleProvider must be placed somewhere else. I'll do it with refactoring of zk list refresh